contracts/truefi/TrueRatingAgency.sol

```diff
@@ -393,7 +406,8 @@ contract TrueRatingAgency is ITrueRatingAgency, Ownable {
             // calculate reward
             // prettier-ignore
             uint256 reward = toTrustToken(interest.mul(
-                distributor.remaining()).div(distributor.amount()));
+                distributor.remaining()).div(distributor.amount()))
+                .mul(rewardMultiplier);
```

Comment:
Maybe it's better to do `.mul(rewardMultiplier)` before `.div(distributor.amount()))` to win a bit of precision

Reply:
Yes, it's always better to divide in the last step.